gordo/machine/dataset/datasets.py

```diff
@@ -231,6 +232,26 @@ def get_data(self) -> Tuple[pd.DataFrame, Optional[pd.DataFrame]]:
         if X.first_valid_index():
             self._metadata["train_start_date_actual"] = X.index[0]
             self._metadata["train_end_date_actual"] = X.index[-1]
+            self._metadata["summary_statistics"] = X.describe().to_dict()
+            hists = {}
+            for tag in X.columns:
+                step = round((X[tag].max() - X[tag].min()) / 100, 6)
+                if step < 9e-07:
+                    hists[str(tag)] = {}
+                    continue
+                outs = pd.cut(
+                    X[tag],
+                    bins=np.arange(
+                        round(X[tag].min() - step, 6),
```

Comment:
Why the +/- step?

Reply:
to include extreme values buckets

Reply:
it is more descriptive than quantiles, make it easier to visualise.